### PR TITLE
Include missing <cstdint> includes

### PR DESCRIPTION
**Pull request by parona-source**
## Description

GCC releases progressively stop implicitly including headers. As such a missing cstdint include is fatal error when building with GCC-15.

https://gcc.gnu.org/gcc-15/porting_to.html

## Motivation and context

https://bugs.gentoo.org/938531

## How has this been tested?

I've built with this patch on top of Gentoo's kodi ebuild, Tests are unaffected and pass as normal.

## Types of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by parona-source** [comment]
Didn't include a component in the commit as this change isn't targeted. I didn't add the header where ever it *could* be needed as it could be included transiently via other header. So this is only getting those I've noticed myself rather than anything smarter like iwyu.

File: xbmc/cores/RetroPlayer/streams/RetroPlayerVideo.h [hunk 1]
```diff
@@ -11,6 +11,8 @@
 #include "IRetroPlayerStream.h"
 #include "cores/RetroPlayer/RetroPlayerTypes.h"
 
+#include <cstdint>
```
Comment (garbear) [inline xbmc/cores/RetroPlayer/streams/RetroPlayerVideo.h:14]: This file (and maybe others) also need `#include <stddef>` for the use of `size_t`. Can you also check for this?

**Comment by garbear** [comment]
I'm OK with targeted includes without using iwyu, they're all fixes individually. And I shudder to think of the mass of headers iwyu would recommend in our codebase :)

**Review by neo1973 — APPROVED** [review]

**Comment by ksooo** [comment]
jenkins build this please

**Comment by neo1973** [comment]
Thanks @parona-source. Can you please also open a pull request against the Omega branch to get this into Kodi 21.2?

**Pull request merged**